lnrpc/lightning.proto

```diff
@@ -3747,9 +3747,14 @@ message PolicyUpdateRequest {
     // The base fee charged regardless of the number of milli-satoshis sent.
     int64 base_fee_msat = 3;
 
-    // The effective fee rate in milli-satoshis. The precision of this value
-    // goes up to 6 decimal places, so 1e-6.
-    double fee_rate = 4;
+    oneof fee {
+        // The effective fee rate in milli-satoshis. The precision of this value
+        // goes up to 6 decimal places, so 1e-6.
+        double fee_rate = 4;
```

Comment:
This breaks the API as is. Instead I think we should just add this as a new field, then switch to the `oneof` later (particularly as we're planning on including this in a minor release).

Reply:
Maybe I'm missing something but moving an existing field into a `oneof` should not break backwards compatibility (see https://developers.google.com/protocol-buffers/docs/proto3#tag_reuse_issues)